packages/zowe-explorer/package.json

```diff
@@ -2224,12 +2224,12 @@
   },
   "dependencies": {
     "@vscode/codicons": "^0.0.35",
-    "@zowe/core-for-zowe-sdk": "^8.28.0",
-    "@zowe/secrets-for-zowe-sdk": "^8.27.4",
-    "@zowe/zos-files-for-zowe-sdk": "^8.28.0",
-    "@zowe/zos-jobs-for-zowe-sdk": "^8.28.0",
-    "@zowe/zosmf-for-zowe-sdk": "^8.28.0",
-    "@zowe/zowe-explorer-api": "3.4.0-SNAPSHOT",
+    "@zowe/core-for-zowe-sdk": "^8.29.4",
+    "@zowe/secrets-for-zowe-sdk": "^8.29.4",
+    "@zowe/zos-files-for-zowe-sdk": "^8.29.4",
+    "@zowe/zos-jobs-for-zowe-sdk": "^8.29.4",
+    "@zowe/zosmf-for-zowe-sdk": "^8.29.4",
+    "@zowe/zowe-explorer-api": "workspace:*",
```

Comment:
Although this change makes sense, I don't think the `workspace:*` version will persist when the Octorelease deploy workflow runs.

Reply:
Ah okay, I can change it back 😋 Note that this was automatically changed from a `pnpm` command and seems to be new to `pnpm@^9`, so we may want to consider handling this within Octorelease?